### PR TITLE
Simpletraj

### PR DESCRIPTION
**Pull request by jan-janssen**

### CODE REVIEW

File: recipes/simpletraj/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (synapticarbors) [inline recipes/simpletraj/meta.yaml]: Could you update the `script` to the latest form found in the `example`:

```
script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

**Comment by synapticarbors** [comment]
@conda-forge/help-python-c any ideas about the failure on Windows?

**Comment by jan-janssen** [comment]
@synapticarbors I guess that is my mistake - simpletraj does not support windows, the same at least it is not available on anaconda from any third party repository https://anaconda.org/search?q=simpletraj

**Comment by synapticarbors** [comment]
linter failure is a bug so it's safe to ignore it:

https://github.com/conda-forge/conda-smithy/issues/876

if you want to skip windows just add:

```
skip: True  # [win]
``` 

in the `build` section.

**Pull request merged**